src/studio/src/designer/frontend/shared/types/index.d.ts

```diff
@@ -149,11 +149,17 @@ export interface IParty {
 }
 
 export interface ILayoutSettings {
-  pages: IPagesSettings;
+  $schema?: string;
+  pages?: IPagesSettings;
+  components?: IComponentsSettings;
 }
 
 export interface IPagesSettings {
-  order: [];
+  order: string[];
```

Comment:
Doesn't this also have an exclude from pdf prop?

Reply:
Jupp. Also a typo ;) Fixed.